client/components/promo-section/promo-card/cta.tsx

```diff
@@ -50,25 +50,28 @@ export interface Props {
 	isPrimary?: boolean;
 }
 
-function isCtaButton( cta: Cta ): cta is CtaButton {
+function isCtaButton( cta: Cta ): boolean {
 	return undefined !== ( cta as CtaButton ).text;
 }
 
-function isCtaAction( action: any ): action is CtaAction {
+function isCtaAction( action: any ): boolean {
```

Comment:
This argument isn't any, it is a union type of `URL | ClickCallback | CtaAction` going by the code above.

Additionally, `action is CtaAction` is a very specific return type that indicates to TS that a given action is a CtaAction. With this code, I see many of the following or similar:
```
Property 'selfTarget' does not exist on type 'string | CtaAction | ClickCallback'.
```